package-system/squish-ccr/Dockerfile.x86_64

```diff
@@ -0,0 +1,54 @@
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# This docker file uses ubuntu 20.04 as the base image so that the AWS Native C++ libraries will use OpenSSL 3 as the base
+# for its dependencies
+#
```

Comment:
The comment should be removed, since this docker file isn't for the AWS C++ SDKK